src/webhook/handlers/choosingReply.js

```diff
@@ -40,6 +42,10 @@ export default async function choosingReply(params) {
     );
   }
 
+  await UserArticleLink.updateTimestamps(userId, data.selectedArticleId, {
```

Comment:
Actually `lastRepliedAt` is not defined as "last reply view date". It is the `createdAt` of latest reply of the article **in the Cofacts database**. It should be updated by cron jobs, choosing a reply should not update the timestamp.

Ref: https://g0v.hackmd.io/eIeU2g86Tfu5VnLazNfUvQ